gamemode/gamemode_client.zig

```diff
@@ -532,9 +532,9 @@ pub var internal_gamemode_client_error_string: [512]u8 = [1]u8{
     0,
 } ++ [1]u8{0} ** 511;
 pub var internal_libgamemode_loaded: c_int = 1;
-pub const api_call_return_int = ?fn () callconv(.C) c_int;
-pub const api_call_return_cstring = ?fn () callconv(.C) [*c]const u8;
-pub const api_call_pid_return_int = ?fn (pid_t) callconv(.C) c_int;
+pub const api_call_return_int = ?*const fn () callconv(.C) c_int;
+pub const api_call_return_cstring = ?*const fn () callconv(.C) [*c]const u8;
+pub const api_call_pid_return_int = ?*const fn (pid_t) callconv(.C) c_int;
```

Comment:
Until the Zig self-hosted compiler stabilizes a bit, I am suggesting others still use the slightly older Zig version: https://github.com/hexops/mach#supported-zig-version

This change would prevent Mach from building on the older version, though. Could you change this PR to have both versions of this file, a `gamemode_client_stage1.zig` and a `gamemode_client.zig` and choose the right one depending on compiler?